ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,203 @@
+    var subtitleTrailingAccessoryView: UIView? { get set }
+    var onTapAction: (() -> Void)? { get set }
+}
+
+/// Properties that make up PersonaView content
+class MSFPersonaViewStateImpl: MSFListCellState, MSFPersonaViewState {
+    init(avatarState: MSFAvatarState) {
+        self.avatarState = avatarState
+    }
+
+    var image: UIImage? {
+        get {
+            return avatarState.image
+        }
+
+        set {
+            avatarState.image = newValue
+        }
+    }
+
+    var primaryText: String? {
+        get {
+            return avatarState.primaryText
+        }
+
+        set {
+            avatarState.primaryText = newValue
+            title = newValue ?? ""
+        }
+    }
+
+    var secondaryText: String? {
+        get {
+            return avatarState.secondaryText
+        }
+
+        set {
+            avatarState.secondaryText = newValue
+            subtitle = newValue ?? ""
+        }
+    }
+
+    var ringColor: UIColor? {
+        get {
+            return avatarState.ringColor
+        }
+
+        set {
+            avatarState.ringColor = newValue
+        }
+    }
+
+    var backgroundColor: UIColor? {
+        get {
+            return avatarState.backgroundColor
+        }
+
+        set {
+            avatarState.backgroundColor = newValue
+        }
+    }
+
+    var foregroundColor: UIColor? {
+        get {
+            return avatarState.foregroundColor
+        }
+
+        set {
+            avatarState.foregroundColor = newValue
+        }
+    }
+
+    var presence: MSFAvatarPresence {
+        get {
+            return avatarState.presence
+        }
+
+        set {
+            avatarState.presence = newValue
+        }
+    }
+
+    var hasPointerInteraction: Bool {
+        get {
+            return avatarState.hasPointerInteraction
+        }
+
+        set {
+            avatarState.hasPointerInteraction = newValue
+        }
+    }
+
+    var isRingVisible: Bool {
+        get {
+            return avatarState.isRingVisible
+        }
+
+        set {
+            avatarState.isRingVisible = newValue
+        }
+    }
+
+    var isTransparent: Bool {
+        get {
+            return avatarState.isTransparent
+        }
+
+        set {
+            avatarState.isTransparent = newValue
+        }
+    }
+
+    var isOutOfOffice: Bool {
+        get {
+            return avatarState.isOutOfOffice
+        }
+
+        set {
+            avatarState.isOutOfOffice = newValue
+        }
+    }
+
+    private var avatarState: MSFAvatarState
+}
+
+/// View for PersonaView
+public struct PersonaView: View {
+    @ObservedObject var state: MSFPersonaViewStateImpl
+    @ObservedObject var tokens: MSFPersonaViewTokens
+
+    public init() {
+        avatar = AvatarView(style: .default, size: .xlarge)
+        state = MSFPersonaViewStateImpl(avatarState: avatar.state)
+        tokens = MSFPersonaViewTokens()
+
+        initializeState()
```

Comment:
You can delete the initializeState func and implement the init() like this:

```
        tokens = MSFPersonaViewTokens()
        let avatar = AvatarView(style: .default,
                                size: .xlarge)
        state = MSFPersonaViewStateImpl(avatarState: avatar.state)
        state.leadingUIView = AnyView(avatar)
        state.leadingViewSize = .xlarge
        state.titleTrailingAccessoryView = state.titleTrailingAccessoryView
        state.subtitleTrailingAccessoryView = state.subtitleTrailingAccessoryView
        state.onTapAction = state.onTapAction
        state.layoutType = .threeLines
```

This will also help you on the removal of the avatar private property.